convert_kinbody_to_urdf.py

```diff
@@ -0,0 +1,194 @@
+# convert_kinbody_to_urdf.py
+
+import os
+import sys
+import xml.etree.ElementTree as ET
+
+
+class Kinbody_Converter:
+    def __init__(self):
+        self.kinbody = None
+        self.urdf = None
+        self.link = None
```

Comment:
Yeah the RViz-pantry_loading thing is what really matters but the other would be a nice sanity check too.